api/test/service/store.test.js

```diff
@@ -414,13 +469,32 @@ test('store/list can be paginated with custom size', async (t) => {
   }
 })
 
+/**
+ * @param {import("@aws-sdk/client-dynamodb").DynamoDBClient} dynamoClient
+ * @param {import("@aws-sdk/client-s3").S3Client} s3Client
+ */
+async function prepareResources (dynamoClient, s3Client) {
```

Comment:
Consider moving `prepareResources`, `createBucket` to `test/helpers/resources,js` and refactoring ``createDynamo*Table` to ``createDynamoTable` that takes a table definition... and extract the 2 table definitions to a single file that can be reused between the infra code and the test code ... we can use the documentDB to get to a closer match... can be a seperate PR tho. Would be good to ensure our infra definition of the table matches our test one